recipes/su2/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "su2" %}
+{% set version = "8.1.0" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/su2code/SU2/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: d550be60ccc8cb907d53a0531715b229585f1e92e2e8e22ab2e8a181d096e7c9
+  patches:
+    - su2-meson.patch
+
+build:
+  skip: True  # [win or osx]
+  number: 0
+  run_exports:
```

Comment:
Does this have a shared library one links to? This isn't tested for in the tests.

Reply:
Is the `run_exports` needed only when there is a shared library ?
In Bioconda it is mandatory for all recipes and I add it here too.
I will remove it!